core/src/components/picker-column/test/picker-column-aria.spec.tsx

```diff
@@ -0,0 +1,47 @@
+import { h } from '@stencil/core';
+import { newSpecPage } from '@stencil/core/testing';
+
+import { PickerColumnCmp } from '../picker-column';
+/**
+ * Stencil has an issue where having multiple spec tests in the same file,
+ * will cause an exception to be thrown. This test is located in a separate file
+ * to avoid this issue.
+ *
+ * Example exception:
+ * ```
+ * TypeError: Cannot read properties of undefined (reading '$instanceValues$')
+    at getValue (/Users/sean/Documents/ionic/framework-next/core/node_modules/@stencil/core/internal/testing/index.js:542:282)
```

Comment:
Can we link to the Stencil issue instead of hardcoding the error?

Reply:
There currently isn't an issue logged to link to. I need to create a reproduction that does the same thing as `ion-picker-column` (using timeouts and executing code within the `connectedCallback`), so the team can diagnose and debug further.

Reply:
Thank you!